projector-server/src/main/kotlin/org/jetbrains/projector/server/idea/UpdatesConfig.kt

```diff
@@ -27,12 +27,12 @@ package org.jetbrains.projector.server.idea
 
 import com.intellij.notification.NotificationDisplayType
 import com.intellij.notification.impl.NotificationsConfigurationImpl
-import org.jetbrains.projector.server.core.ij.invokeWhenIdeaIsInitialized
+import org.jetbrains.projector.util.loading.state.invokeWhenIdeaIsAtState
 import org.jetbrains.projector.util.loading.UseProjectorLoader
 
 fun forbidUpdates() {
   forbidPlatformUpdates()
-  invokeWhenIdeaIsInitialized("Forbid platform updates and plugin update notifications") {
+  invokeWhenIdeaIsAtState("Forbid platform updates and plugin update notifications") {
```

Comment:
Minor: this is a bit unclear when a default parameter is used... Maybe get rid of the default parameter and always specify the needed state?

Reply:
I thought I asked about new naming but turns out it was just my thought.. If you don't have any suggestions on better naming I'll remove default parameter